app/helpers/passengers_helper.rb

```diff
@@ -19,11 +19,9 @@ def passengers_table_row_class(passenger)
     doctors_note_row_class(passenger.doctors_note)
   end
 
-  def passengers_table_class
-    if @current_user.admin?
-      'row-border admin-table'
-    else
-      'row-border dispatch-table'
+  def sortable_date(note)
+    if note.present?
+      note.expiration_date.strftime('%Y%m%d%H%M%s')
```

Comment:
Perhaps a little precise for an expiration "date".  Datatables supports "secondary sort", but it doesn't work if two things that _look_ identical are actually different.

Reply:
That's what I get for copying and pasting off the internet.